spec/controllers/steps/prefill_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Steps::PrefillController, :user_session do
+  let(:user_session) { build(:user_session) }
+
+  describe 'GET #show' do
+    subject(:response) { get :show, params: params }
+
+    let(:params) do
+      {
+        commodity_code: '0702000007',
+        country_of_origin: 'FI',
+        import_date: '2021-02-17',
+        import_destination: 'UK',
+        redirect_to: 'http://localhost/flibble',
+      }
+    end
+
+    before do
+      allow(UserSession).to receive(:build_from_params).and_call_original
+    end
+
+    it 'builds the session from params' do
+      response
```

Comment:
what about renaming `response` with an "action"?
it sounds strange just to call "response"

possible name: call_show, perform_show, do_something, .... up to you

Reply:
I think we could but we'd be going against a common pattern in the app. Maybe something for a separate refactor as it's much worse to use different patterns in the same app than getting one that you feel is right and another you feel is wrong or do a full refactor in the same PR and end up with unrelated changes.